omod/src/main/webapp/resources/htmlforms/003-HMIS_HIVCare_ARTCard-ClinicalAssessmentPage.xml

```diff
@@ -0,0 +1,1531 @@
+                    if (viral_load_status == 1306) {
+                        /* Not Detected */
+                        disable_enable_fields(["viral-load-results-copies"], false);
+                        setValue('viral-load-results-copies.value', 1);
+                    }
+                });
+
+                jq("#viral-load-results-copies").change(function () {
+                previousVl = jq("#viral-load-results-copies").find("input").val();
+                stabilityCheck();
+                });
+
+                // trigger a change when the form loads
+                jq("#viral-load-results-qualitative").change();
+
+                /*determine viral load status*/
+
+                jq("#viral-load-results-copies").keyup(function () {
+                    if (jq("#viral-load-results-copies").find("input").val() &lt; 1000)
+                        jq("#viral-load-status").val("Suppressed");
+                    else if (jq("#viral-load-results-copies").find("input").val() >= 1000)
+                        jq("#viral-load-status").val("Non-suppressed");
+                    else
+                        jq("#viral-load-status").val(" ");
+                });
+
+                /*enable disable Art adherence*/
+
+                jq(".EnableDisableAdherenceReason").find("input[type$='checkbox']").attr('disabled', true);
+                //jq(".EnableDisableAdherenceReason").addClass("html-form-entry-disabled-field");
+
+                jq(".disableARTAdherence").change(function () {
+                    var adherence = jq(this).find("input[type='radio']:checked").val();
+                    if(adherence == 90157 || adherence == 90158) {
+                        jq(".EnableDisableAdherenceReason").find("input[type$='checkbox']").attr('disabled', false);
+                        //jq(".EnableDisableAdherenceReason").removeClass("html-form-entry-disabled-field");
+                    } else {
+                        jq(".EnableDisableAdherenceReason").find("input[type$='checkbox']").attr('disabled', true);
+                        //jq(".EnableDisableAdherenceReason").addClass("html-form-entry-disabled-field");
+                    }
+                });
+
+                /*enable disable Hepatitis B status*/
+
+                jq(".EnableDisableHepBStatus").find("input[type$='radio']").attr('disabled', true);
+
+                jq(".HepBStatus").change(function () {
+                    var hepStatus = jq(this).find("input[type='radio']:checked").val();
+                    if(hepStatus == 159971) {
+                        jq(".EnableDisableHepBStatus").find("input[type$='radio']").attr('disabled', false);
+                    } else {
+                        jq(".EnableDisableHepBStatus").find("input[type$='radio']").attr('disabled', true);
+                    }
+
+                });
+
+                jq("#bmi").find("input[type$='text']").attr('disabled', true);
+
+                jq(".enableDisable").each(function () {
```

Comment:
@semujju we had agreed to remove these legacy functions

Reply:
done